compiler/infra/Assert.cpp

```diff
@@ -44,13 +44,6 @@ void OMR_NORETURN TR::trap()
    static char * noDebug = feGetEnv("TR_NoDebuggerBreakPoint");
    if (!noDebug)
       {
-      static char *crashLogOnAssume = feGetEnv("TR_crashLogOnAssume");
-      if (crashLogOnAssume)
-         {
-         //FIXME: this doesn't work on z/OS
-         *(volatile int*)(0) = 0; // let crashlog do its thing
-         }
-
 #ifdef _MSC_VER
```

Comment:
@babsingh yes it will run on Windows. There is an ifdef to handle Windows specifically.

Reply:
LTGM: `DebugBreak` on Windows and `raise(SIGTRAP)` on Unix platforms.